workflow-runtime/build.gradle.kts

```diff
@@ -18,35 +14,51 @@ java {
 
 apply(from = rootProject.file(".buildscript/configure-maven-publish.gradle"))
 
-// Benchmark configuration.
-configure<JMHPluginExtension> {
-  include = listOf(".*")
-  duplicateClassesStrategy = DuplicatesStrategy.WARN
-}
-configurations.named("jmh") {
-  attributes.attribute(Usage.USAGE_ATTRIBUTE, objects.named(Usage.JAVA_RUNTIME))
-}
-tasks.named<KotlinCompile>("compileJmhKotlin") {
-  kotlinOptions {
-    // Give the benchmark code access to internal definitions.
-    val compileKotlin: KotlinCompile by tasks
-    freeCompilerArgs += "-Xfriend-paths=${compileKotlin.destinationDir}"
+kotlin {
+  jvm {
+    compilations {
+      val main by getting
+      val workflowNode by creating {
+        kotlinOptions {
+          val compileKotlinJvm: KotlinCompile by tasks
+          freeCompilerArgs += "-Xfriend-paths=${compileKotlinJvm.destinationDir}"
```

Comment:
if this isn't working do we still need it?

Reply:
It works across compilations within a module, but not across modules.